apps/src/applab/designMode.js

```diff
@@ -415,6 +415,9 @@ designMode.updateProperty = function (
       } else if (DATA_PREFIX_REGEX.test(value)) {
         element.src = value;
       } else {
+        if (!getStore().getState().pageConstants.isCurriculumLevel) {
```

Comment:
Does this code path only get hit when remixing? It sounds pretty general (`designMode.updateProperty`)?

Reply:
It also gets hit when sharing as well - so if a user is logged in and then shares the link to the project level, the images will be 'broken' which is ideal because if 'image://' was removed from the image file name, then the image would be broken in the original level.

Reply:
I'll add more details about this in the PR description above.

Reply:
This feel like a change that might go here (looks like this function gets called soon after your current change), is there a reason it isn't?

https://github.com/code-dot-org/code-dot-org/blob/a2624d39e24d257f49ab5584b02e651793f02a3d/apps/src/assetManagement/assetPrefix.js#L86-L92

I'm also wondering if there's a more general case that this applies to (ie, could it be "all standalone projects" rather than "non-curriculum levels", or would that break things?)

Reply:
I thought about placing logic here. But when `fixPath` is called, if 'image://' is prepended to file name and the level is not a curriculum level, then it satisfies condition at line 86 and the wrong filename path is returned. It needs to be removed before `fixPath` is called so its file path is determined by line 101.

Reply:
From my understanding, the 'image://' path prefix is added only when images are stored as starter assets. But I'm not sure exactly why.

Reply:
Could you put your new condition at line 85 and return it before you hit the existing condition on line 86? Seems very much like what you're doing is "fixing the path", yeah?

Reply:
Oh yeah! Silly me. This definitley makes more sense to place logic in `assetPrefix.js`. Thanks os much!

Reply:
Hey again - I made the change you suggested and saw on the [Drone build](https://drone.cdn-code.org/code-dot-org/code-dot-org/32943/1/4) there are several additional unit test failures. I definitely do not mind going back and fixing them as I did in `designModeTest.js` and `importTest.js`. But it did make me wonder about the placement of the `isCurriclumLevel` check. I wonder if we want to scope the check for starter asset images in the App Lab Design mode by including it in only `designMode.js`. Thoughts?

Reply:
@bencodeorg and I discussed offline and agree that it makes sense to keep logic in `assetPrefix.js`.